.github/workflows/ci_jruby.yml

```diff
@@ -119,3 +119,47 @@ jobs:
       - name: Annotate errors
         if: ${{ failure() }}
         uses: ./.github/actions/annotate
+  
+  notify_slack_fail:
```

Comment:
I wonder if it's worth adding a custom action that holds the `notify_slack_fail` and `notify_slack_success` logic in it, like the refactor Tanna did for the multiverse. 

Not something that needs to be done as part of this PR, just something to consider down the line.